### ISSUE TITLE
aflplusplus_optimal

### ISSUE THREAD

**Issue by hpathuri** [google/fuzzbench#640]
It is unclear which aflplusplus version has the best ranking in Fuzz Bench testing? Is there a particular version that we need to use or the latest?

**Comment by oliverchang** [comment]
Moved this issue seems it seems like a FuzzBench specific question.

**Comment by jonathanmetzman** [comment]
@vanhauser-thc should know.
I think the config aflplusplus_optimal is the best.
Note that I don't think the different "versions" of AFL++ are different versions of the source code. I think it's mostly the same source but configured differently.

**Comment by hpathuri** [comment]
Thanks. It would be good to understand this optimal configuration so that everyone can benefit. Is there a blogpost with these details? Any help is greatly appreciated.

**Comment by jonathanmetzman** [comment]
No idea about a blogpost. But if you want to know how to use this config without really understanding it, you can read fuzzers/aflplusplus_optimal/fuzzer.py

**Comment by vanhauser-thc** [comment]
@hpathuri we rewrote the main README and it is very clear now for which option exists and which are usually the best.
TLDR: use the newest llvm you can, try afl-clang-lto and use afl-clang-fast otherwise. AFL_LLVM_CMPLOG is usually the best, sometimes AFL_LLVM_LAF_ALL is better. run time options do not need anything special usually.

@jonathanmetzman we talked about which option variants we would propose for oss-fuzz. where would you like me to put these down? maybe an issue at oss-fuzz?

**Comment by jonathanmetzman** [comment]
> @jonathanmetzman we talked about which option variants we would propose for oss-fuzz. where would you like me to put these down? maybe an issue at oss-fuzz?

That sounds good to me. Thank you for your help!
